Assets/_BForBoss/_Utility/Scripts/PGInputSystem.cs

```diff
@@ -29,91 +28,125 @@ public class PGInputSystem
         public event Action<bool> OnScrollWeaponChangeAction;
         public event Action<int> OnDirectWeaponChangeAction;
         public event Action OnPausePressed;
-        public event Action OnUnpausePressed;
 
         public PGInputSystem(InputActionAsset asset)
         {
             _actionAsset = asset;
             SetupActionMapInput();
+            SetupFindActions();
             SetupPlayerActions();
             SetupUIActions();
         }
 
         public void SetToUIControls()
         {
-            _UIControlsActionMap.Enable();
-            _playerControlsActionMap.Disable();
+            _UIControlsActionMap?.Enable();
+            _playerControlsActionMap?.Disable();
             LockMouseUtility.Instance.UnlockMouse();
         }
         
         public void SetToPlayerControls()
         {
-            _UIControlsActionMap.Disable();
-            _playerControlsActionMap.Enable();
+            _UIControlsActionMap?.Disable();
+            _playerControlsActionMap?.Enable();
             LockMouseUtility.Instance.LockMouse();
         }
+
+        public void ForceUnbind()
```

Comment:
This is the main thing I don't like, but it's just Unity being unity. 

Written this test for the sole purpose of unit test, this is not good practice.